### DISCUSSION TITLE
Update Meilisearch

### DISCUSSION THREAD

**Discussion by Twilek-de** [karakeep-app/karakeep#751] [Q&A]
I have Hoarder running with a Meilisearch V1.6. The current version is V1.10.1. If I update the container I get an error that the database version and the version of the executable are not the same. Can I update the database or can I simply delete the database and have Meilisearch reindex itself?

**Comment by MohamedBassem** [comment]
Meilisearch is not backward compatible, so you'll have to delete its database, and invoke a "Reindex" from the admin panel. Re-indexing is typically quite fast.

**Comment by Twilek-de** [comment]
Thank you! No problem, I just didn´t want to bork the whole installation.

**Reply by MohamedBassem** [reply]
Just to be clear, you only need to wipe Meillisearch's database, not the entire installation.

**Reply by Twilek-de** [reply]
I have a directory mapped in the compose file for the Meilisearch data. I would stop the containers, update the compose file for the new meilisearch version, empty that directory and restart the containers with the new meilisearch version and trigger a re-indexing.

**Reply by MohamedBassem** [reply]
Yeah, that should work.

**Reply by Twilek-de** [reply]
Worked without a problem....
